examples/auth.py

```diff
@@ -15,6 +15,30 @@ async def run():
         obo_auth_session = bdk.obo(username="username")
         logging.info(await obo_auth_session.session_token)
 
+        # update private key and certificate
+        private_key_string = '-----BEGIN RSA PRIVATE KEY-----\n'\
```

Comment:
As of today we don't support support certificate authentication. So please remove the certificate examples from the example.